src/components/FormFieldToggle.jsx

```diff
@@ -0,0 +1,47 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import ToggleButton from 'react-toggle-button'
+import elementStyles from '../styles/isomer-cms/Elements.module.scss';
+
+const creatableSelectHandler = (callback, dropdownEvent, id) => {
+    callback({
+        target: {
+            id,
+            value: dropdownEvent,
+        },
+    })
+}
+
+const FormFieldDropdown = ({
+    title,
+    value,
+    id,
+    onFieldChange,
+}) => (
+    <>
+      <div className={elementStyles.formHorizontal}>
+        <div className={`d-flex align-items-center ${elementStyles.formHorizontalLabel}`}>
+            <span>{`${title}:`}</span>
+        </div>
+        <div className={`d-flex align-items-center ${elementStyles.formHorizontalLabel}`}>
+            <ToggleButton
+                className={elementStyles.formHorizontalInput}
+                inactiveLabel={''}
+                activeLabel={''}
+                thumbIcon={<span className={`bx bx-xs ${value ? `${elementStyles.formToggleThumbIconEnabled} bx-check` : `${elementStyles.formToggleThumbIconDisabled} bx-x`}`}/>}
+                value={value}
+                onToggle={(value) => creatableSelectHandler(onFieldChange, !value, id)}
+            />
+        </div>
+      </div>
+    </>
+);
+  
+export default FormFieldDropdown;
```

Comment:
should be `FormFieldToggle`

Reply:
Good catch, resolved in https://github.com/isomerpages/isomercms-frontend/pull/321/commits/257e58a58af8a4f1e2a868f2ce5fdfa23a2b453c